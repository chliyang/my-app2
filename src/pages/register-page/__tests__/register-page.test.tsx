import renderer from "react-test-renderer";
import RegisterPage from "../register-page";

describe("RegisterPage", () => {
  it("should render register page", () => {
    const registerPageDom = renderer.create(<RegisterPage />).toJSON();
    expect(registerPageDom).toMatchSnapshot();
  });
});
