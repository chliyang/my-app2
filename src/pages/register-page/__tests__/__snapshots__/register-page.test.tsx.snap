// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterPage should render register page 1`] = `
<div
  className="register"
>
  <form
    className="ant-form ant-form-horizontal register-form"
    id="normal_register"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <h1
      className="title"
    >
      注册信息填写
    </h1>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-label ant-col-xs-4 ant-col-sm-6"
        role="cell"
        style={Object {}}
      >
        <label
          className="ant-form-item-required"
          htmlFor="normal_register_name"
          title="用户名"
        >
          用户名
        </label>
      </div>
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <input
              className="ant-input"
              id="normal_register_name"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="请设置您的用户名"
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-label ant-col-xs-4 ant-col-sm-6"
        role="cell"
        style={Object {}}
      >
        <label
          className="ant-form-item-required"
          htmlFor="normal_register_email"
          title="邮箱"
        >
          邮箱
        </label>
      </div>
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <input
              className="ant-input"
              id="normal_register_email"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="请输入邮箱地址"
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-label ant-col-xs-4 ant-col-sm-6"
        role="cell"
        style={Object {}}
      >
        <label
          className="ant-form-item-required"
          htmlFor="normal_register_password"
          title="密码"
        >
          密码
        </label>
      </div>
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <span
              className="ant-input-affix-wrapper ant-input-affix-wrapper-has-feedback"
              onMouseDown={[Function]}
            >
              <input
                className="ant-input"
                hidden={null}
                id="normal_register_password"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                placeholder="请设置您的密码"
                style={null}
                type="password"
                value=""
              />
              <span
                className="ant-input-suffix"
              />
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-label ant-col-xs-4 ant-col-sm-6"
        role="cell"
        style={Object {}}
      >
        <label
          className="ant-form-item-required"
          htmlFor="normal_register_confirm"
          title="确认密码"
        >
          确认密码
        </label>
      </div>
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <span
              className="ant-input-affix-wrapper ant-input-affix-wrapper-has-feedback"
              onMouseDown={[Function]}
            >
              <input
                className="ant-input"
                hidden={null}
                id="normal_register_confirm"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                placeholder="请二次确认您的密码"
                style={null}
                type="password"
                value=""
              />
              <span
                className="ant-input-suffix"
              />
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-label ant-col-xs-4 ant-col-sm-6"
        role="cell"
        style={Object {}}
      >
        <label
          className="ant-form-item-required"
          htmlFor="normal_register_phone"
          title="电话号码"
        >
          电话号码
        </label>
      </div>
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <input
              className="ant-input"
              id="normal_register_phone"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="请输入您的电话号码"
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-row ant-form-item"
      role="row"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16"
        role="cell"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <button
              className="ant-btn ant-btn-primary login-form-button"
              onClick={[Function]}
              type="submit"
            >
              <span>
                立即注册
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
    <div />
  </form>
</div>
`;
